.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+### Description
+[Describe what this change achieves.]
+ 
+### Issues Resolved
+[List any issues this PR will resolve. You should likely open an issue if one does not already exist.]
+ 
+### Check List
+- [ ] Commits are signed per the DCO using --signoff
+
+For any changes to files within Helm chart directories:
+- [ ] Helm chart version bumped
+- [ ] Helm chart `CHANGELOG.md` updated to reflect change
+
+By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
+For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
```

Comment:
This is pointing to OpenSearch can you point it to the on in helm repo or just use.
https://github.com/opensearch-project/helm-charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin

Reply:
Done, thanks.